include/deal.II/numerics/vector_tools_boundary.templates.h

```diff
@@ -219,7 +218,7 @@ namespace VectorTools
             q_collection,
             update_quadrature_points);
 
-          typename DoFHandlerType<dim, spacedim>::active_cell_iterator
+          typename DoFHandler<dim, spacedim>::active_cell_iterator
             cell = dof.begin_active(),
             endc = dof.end();
           for (; cell != endc; ++cell)
```

Comment:
Can we have a range-based for loop here?

Reply:
Done!